lucifer/Shell/_Info.py

```diff
@@ -0,0 +1,14 @@
+def print_id(self, *args, **kwargs):
+    print(f"Shell ID: {self.id}")
+
+
+def print_name(self, *args, **kwargs):
+    print(f"Shell Name: {self.name}")
+
+
+def help(self, *args, **kwargs):
```

Comment:
Codacy found an issue: [Redefining built-in 'help'](https://app.codacy.com/manual/alexspam9090/Lucifer/pullRequest?prid=6170630)